### ISSUE TITLE
Passenger Preference Pane needs to update the hosts database.

### ISSUE THREAD

**Issue by jaredmoody** [Fingertips/passengerpane#42]
For some reason I'm now getting this message every time I launch the pane, and when I add a new app, it doesn't create the conf file under passenger_pane_vhosts anymore either.

I tried removing the rubycocoa I had at /Library/Frameworks and reinstalling the latest version, but no luck.  

What else could I try?  Thanks!

**Comment by alloy** [comment]
Could you try to figure out why the allApplicationHostsExist? method returns true in your situation and if it's valid or not?

PassengerApplication#allApplicationHostsExist? http://github.com/alloy/passengerpane/blob/master/app/models/passenger_application.rb#L35

Called from the controller at:
http://github.com/alloy/passengerpane/blob/master/app/controllers/passenger_pref.rb#L274

**Comment by jaredmoody** [comment]
it seems the allApplicationHostsExist? is valid, because /usr/bin/dscl localhost -list /Local/Default/Hosts isn't listing all the apps shown in the pane

To test this and try and reset everything, I did the following:
- removed passenger_pane_vhosts directory from /etc/apache2/
- removed include of that dir from /etc/apache2/httpd.conf
- ran /usr/bin/dscl localhost -delete /Local/Default/Hosts to clear that list
- removed and reinstalled the pane

The pane then came up clear, but when I added a new app, it didn't update the httpd.conf, didn't create the passenger_pane_vhosts directory or the vhost file, and the output of /usr/bin/dscl localhost -list /Local/Default/Hosts didn't change either.

Thanks for helping me figure this out, I do appreciate it.
